### PR TITLE
Rpm/foreman tasks 0.10.9 [rpm/1.16]

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by iNecas** [comment]
Needed to fix remote execution and katello's notifications in 1.16

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 1]
```diff
@@ -17,21 +17,20 @@
...
+Release: 1%{?foremandist}%{?dist}
 Group: Development/Libraries
 License: GPLv3
 URL: https://github.com/theforeman/foreman-tasks
 Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: foreman >= 1.15.0
 
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core)
-Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.22
+Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.29
```
Comment (ekohl) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: I think this also needs a package update:
```DEBUG util.py:439:  Error: No Package found for tfm-rubygem(dynflow) >= 0.8.29```

Comment (iNecas) [reply rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: This was supposed do be done already here https://github.com/theforeman/foreman-packaging/pull/1983. Is there anything else I have to do?

Comment (ekohl) [reply rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Looks like I only released it to plugins (nonscl) and not core foreman (scl). I'll fix that later.

**Comment by ekohl** [comment]
[test]

**Pull request merged**